File: app/src/main/java/com/example/recyclerviewdbmultilayout/Movie.java
package com.example.recyclerviewdbmultilayout;

public class Movie {

    private String title;
    private String descripition;

    public Movie(String title, String descripition) {
        this.title = title;
        this.descripition = descripition;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescripition() {
        return descripition;
    }

    public void setDescripition(String descripition) {
        this.descripition = descripition;
    }
}

File: app/src/main/java/com/example/recyclerviewdbmultilayout/Movie.kt
package com.example.recyclerviewdbmultilayout

class Movie(var viewType: Int, var title: String, var descripition: String, var photo: Int) {

    companion object {
        //Add the type indicators here
        const val COMMEDY_MOVIE_TYPE = 0
        const val LOVE_STORY_MOVIE_TYPE = 1
    }
}